### PR TITLE
Roles logic

### PR DESCRIPTION
**Pull request by halawami**
Rebase with updated gh-pages

### CODE REVIEW

File: src/endpoints.js [hunk 1]
```diff
@@ -16,9 +16,41 @@ function messageHandler(message){
 		let member = message.mentions.members.first() || message.guild.members.get(args[0]);
 		if(!member)
 		  return message.reply("Please mention a valid member of this server");
-			// TODO: call democracy function
+		// TODO: call democracy function
 	}	
 
+	if (command === "add-role") {
```
Comment (CelticMajora) [inline src/endpoints.js]: Will these two just be for debugging

Comment (halawami) [reply src/endpoints.js]: Just as proof of concept for Discord bot instead of removing/adding people.

**Comment by amgassert** [comment]
#23 also covers logic for roles. I'd argue that splitting that stuff out into another file is much cleaner than squeezing everything into the endpoints file.

**Comment by CelticMajora** [comment]
I think we should sort out responsibilities between the two PRs before merging either

**Comment by CelticMajora** [comment]
Also, why is this not being merged into gh-pages?

**Comment by halawami** [comment]
I think we can get rid of this branch/take the things we need from it and add it to #23? What do you think @CelticMajora @amgassert ?

**Pull request closed without merging**